demos/dqx_demo_asset_bundle/databricks.yml

```diff
@@ -29,12 +35,14 @@ resources:
               maintenance_rules_file: "${var.maintenance_rules_file}"
               sensor_rules_file: "${var.sensor_rules_file}"
             source: WORKSPACE
-          environment_key: Default
+      job_clusters:
+        - job_cluster_key: dqx_demo_cluster
```

Comment:
can we use serverless cluster so that we don't have to maintain the cluster config?